File: HoWo5_6/PhoneCompany/Denzel.java
package HoWo5_6.PhoneCompany;

import HoWo5_6.Phone.Phone;

public class Denzel extends Phone{



    public Denzel(String model, double sizeDisplay, int akb, int price) {
        super(model, sizeDisplay, akb, price);
    }

    @Override
    public void charge() {
        System.out.println("опять эта рухлядь разрядилась");
    }

    @Override
    public String toString() {
        return "Denzel ["+ super.toString()+"]";
    }


}

File: HoWo7/PhoneCompany/Denzel.py
from Phone.Phone import Phone

class Denzel (Phone):

    def __init__(self, model:str,sizeDisplay,akb:int,price:int):
        super().__init__(model, sizeDisplay, akb, price)

    def charge(self):
        print("опять эта рухлядь разрядилась")

    def __str__(self)->str:
        return f"Denzel [{super().__str__()}]"